cluster-provision/images/vm-image-builder/README.md

```diff
@@ -40,16 +43,30 @@ then publish it to a registry. In this case a local registry:
 publish-containerdisk.sh example localhost:1234/myimage:mytag
 ```
 
+To build `example` for arm64(aarch64), only need to set the environmental
+variable.
+```
+export ARCHITECTURE=arm64
+```
+
+To build the Virtual Machine without console output, only need to set the
+environmental variable CONSOLE. This is useful when the building script
+run in a CICD pipeline.
+```
+export CONSOLE=no
+```
+Then follow the previous step.
 ## Create a new containerdisk
 
 Every directory contains build instructions for virt-customize. The build
 instructions are distributed between the following three files:
 
 ```
-$ ls -1 example/
-cloud-config #cloud-init configuration for virt-customize
-image-url    #download URL of the base image
-os-variant   #operating system variant (for example fedora32)
+$ ls -l example/
+cloud-config		# cloud-init configuration for virt-customize
+image-url		# download URL of the base image
+image-url-arm64		# download URL of the base image for Arm64
+os-variant		# operating system variant (for example fedora32)
```

Comment:
These are still not aligned O:-) Can you please just replace the tabs with spaces? That should solve it once and for all :)